src/test/java/com/codejoust/main/api/GameTests.java

```diff
@@ -208,7 +209,6 @@ public void runCodeSuccess() throws Exception {
         assertEquals(TestFields.OUTPUT, resultDto.getUserOutput());
         assertNull(resultDto.getError());
         assertEquals(TestFields.INPUT, resultDto.getInput());
-        assertEquals(TestFields.OUTPUT, resultDto.getCorrectOutput());
```

Comment:
nit: Is this assertion unused now - is there some reason for removing this line? Is the output hidden, and if so, could that be tested instead?